docs/installation.rst

```diff
@@ -0,0 +1,118 @@
+.. meta::
+   :description: HSA runtime implementation
+   :keywords: ROCR, ROCm, library, tool, runtime
+
+.. _installation:
+
+====================
+Installation
+====================
+
+This document provides information required to build and install ROCR using prebuilt binaries or from source.
+
+Build and install using prebuilt binaries
+-------------------------------------------
+
+Here is how you can install ROCR using prebuilt binaries.
+
+Prerequisites
```

Comment:
@dayatsin-amd Add commands wherever applicable under Prerequisites.

Reply:
@dayatsin-amd I have already added "See `how to install ROCm <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/>`_."
Should we require command to install ``libdrm`` package?